pages/apis/rest_api/builds.md

```diff
@@ -89,6 +89,10 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pipeline.
         "type": "script",
         "name": ":package:",
         "step_key": "package",
+        "step": {
+          "id": "018c0f56-c87c-47e9-95ee-aa47397b4496",
+          "signature": null
```

Comment:
it might be nice to add an example of the signature here - here's one from a recent agent build
```json
{
  "signature": {
    "value": "eyJhbGciOiJFUzI1NiIsImtpZCI6InlvdSBzbHkgZG9nISB5b3UgY2F1Z2h0IG1lIG1vbm9sb2d1aW5nISJ9..gcmV8TgY6MfAXRbD_7kyuB3JAQMH-0th5E1MmQn5ND34bEs-zNBZ3tLi6w2e_gvfqL6C1kxRhOQzAwEZMQlo5w",
    "algorithm": "HS512",
    "signed_fields": [
      "command",
      "env",
      "matrix",
      "plugins",
      "repository_url"
    ]
  }
}
```